config/management/src/storage_helper.rs

```diff
@@ -0,0 +1,212 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{error::Error, Command};
+use libra_crypto::ed25519::Ed25519PublicKey;
+use libra_global_constants::{
+    ASSOCIATION_KEY, CONSENSUS_KEY, EPOCH, FULLNODE_NETWORK_KEY, LAST_VOTED_ROUND, OPERATOR_KEY,
+    OWNER_KEY, PREFERRED_ROUND, VALIDATOR_NETWORK_KEY, WAYPOINT,
+};
+use libra_network_address::NetworkAddress;
+use libra_secure_storage::{NamespacedStorage, OnDiskStorage, Policy, Storage, Value};
+use libra_types::{account_address::AccountAddress, transaction::Transaction};
+use std::fs::File;
+use structopt::StructOpt;
+
+pub struct StorageHelper {
+    temppath: libra_temppath::TempPath,
+}
+
+impl StorageHelper {
+    pub fn new() -> Self {
+        let temppath = libra_temppath::TempPath::new();
+        temppath.create_as_file().unwrap();
+        File::create(temppath.path()).unwrap();
+        Self { temppath }
+    }
+
+    pub fn storage(&self, namespace: String) -> Box<dyn Storage> {
+        let storage = OnDiskStorage::new(self.temppath.path().to_path_buf());
+        Box::new(NamespacedStorage::new(storage, namespace))
+    }
+
+    pub fn path_string(&self) -> &str {
+        self.temppath.path().to_str().unwrap()
+    }
+
+    pub fn initialize(&self, namespace: String) {
+        let policy = Policy::public();
+        let mut storage = self.storage(namespace);
+
+        storage.create_key(ASSOCIATION_KEY, &policy).unwrap();
+        storage.create_key(CONSENSUS_KEY, &policy).unwrap();
+        storage.create_key(FULLNODE_NETWORK_KEY, &policy).unwrap();
+        storage.create_key(OWNER_KEY, &policy).unwrap();
+        storage.create_key(OPERATOR_KEY, &policy).unwrap();
+        storage.create_key(VALIDATOR_NETWORK_KEY, &policy).unwrap();
+
+        storage.set(EPOCH, Value::U64(0)).unwrap();
+        storage.set(LAST_VOTED_ROUND, Value::U64(0)).unwrap();
+        storage.set(PREFERRED_ROUND, Value::U64(0)).unwrap();
+        storage.set(WAYPOINT, Value::String("".into())).unwrap();
+    }
+
+    pub fn association_key(
+        &self,
+        local_ns: &str,
+        remote_ns: &str,
+    ) -> Result<Ed25519PublicKey, Error> {
+        let args = format!(
+            "
+                management
+                association-key
+                --local backend={backend};\
+                    path={path};\
+                    namespace={local_ns}
+                --remote backend={backend};\
+                    path={path};\
+                    namespace={remote_ns}\
+            ",
+            backend = crate::secure_backend::DISK,
```

Comment:
nit: why not import directly?

Reply:
same, plus I guess I kinda see constants more in line with functions